authz/sdk_server_interceptors_test.go

```diff
@@ -43,14 +67,51 @@ func TestNewStatic(t *testing.T) {
 					}
 				]
 			}`,
-			wantErr: false,
 		},
 	}
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			if _, err := authz.NewStatic(test.authzPolicy); (err != nil) != test.wantErr {
+			if _, err := authz.NewStatic(test.authzPolicy); (err != nil) && (err.Error() != test.wantErr) {
```

Comment:
Notice that this does not properly fail if `err == nil && test.wantErr != ""`.

Probably what I'd do is define `wantErr` as `error` and then do:
```go
if _, err := authz.NewStatic(test.authzPolicy); fmt.Sprint(err) != fmt.Sprint(test.wantErr) {
```

Reply:
Oh right! Updated code. Thank you!